rebalance/rebalance.py

```diff
@@ -314,10 +315,11 @@ def get_ideal_ratio(channels: list, enough_liquidity: Millisatoshi):
 
 
 def feeadjust_would_be_nice(plugin: Plugin):
-    try:
+    commands = [c for c in plugin.rpc.help().get("help") if c["command"].split()[0] == "feeadjust"]
```

Comment:
Thx, nice!

We should add this a generic method for this to pyln rpc code, that calls 'help' just the first time and caches it. Then we could have something like this:

```python
if plugin.has_method("somefunction"):
    # foo
```

Reply:
See https://github.com/lightningd/plugins/pull/201

Reply:
Update: doesn't seem to work, as rpc code generates method on the fly when getattr is called.